x/pairing/keeper/pairing.go

```diff
@@ -246,6 +246,9 @@ func (k Keeper) CalculateEffectiveAllowedCuPerEpochFromPolicies(policies []*plan
 	var policyTotalCuLimit []uint64
 	for _, policy := range policies {
 		if policy != nil {
+			if policy.EpochCuLimit == 0 || policy.TotalCuLimit == 0 {
```

Comment:
should be 2 if cases, one for each limit

Reply:
done. see https://github.com/lavanet/lava/commit/a85725d29f3c84baa38732c19018a9970ae7bca3